src/components/Expandable.tsx

```diff
@@ -5,6 +5,7 @@ import 'components/styles/Expandable.scss'
 
 export const ControlledExpandable: React.FunctionComponent<{
   id: string
+  noContract?: boolean
```

Comment:
Perhaps this should be "clickable"? 

Prefer using boolean to optional<boolean>. Optional<boolean> implies three states, and doesn't imply what the default behavior is.

Reply:
A more common/utilitarian solution might be to optionally take an onClick function which overrides the standard onClick.